### PR TITLE
copy stuffs to clipboard

### PR DESCRIPTION
**Pull request by BenjaminCall**
# Code-review checklist #

  * Documentation
      * [ ] Interfaces describe how other programmers would use them.
      * [ ] Code that is not obvious has documentation describing why
        that code is the way it is.

### CODE REVIEW

File: client/js/_author/components/bank_navigation/bank_list_item.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React      from 'react';
 import _          from 'lodash';
+import CopyToClipboard  from 'react-copy-to-clipboard';
```
Comment (BenHSmith13) [inline client/js/_author/components/bank_navigation/bank_list_item.jsx]: misaligned

File: client/styles/_author/partials/_dashboard.scss [hunk 2]
```diff
@@ -178,15 +178,51 @@
...
+  }
+
+  &:hover{
+    background-color: $white;
+
+    i{
+      color: $purple;
+    }
+  }
+}
```
Comment (BenHSmith13) [inline client/styles/_author/partials/_dashboard.scss]: newLine

File: package.json [hunk 3]
```diff
@@ -29,8 +28,10 @@
     "react, qti, assessments, quizzes"
   ],
   "dependencies": {
-    "del": "^2.2.2"
+    "clipboard": "^1.6.0",
```
Comment (BenHSmith13) [inline package.json]: This is in the wrong package.json

File: package.json [hunk 4]
```diff
@@ -29,8 +28,8 @@
     "react, qti, assessments, quizzes"
   ],
   "dependencies": {
-    "del": "^2.2.2"
+    "del": "^2.2.2",
+    "react": "^15.4.2"
```
Comment (BenHSmith13) [inline package.json]: dont want react in here either

Comment (BenjaminCall) [inline package.json]: how the hell are these things getting in there...
i will fix it

**Comment by BenHSmith13** [comment]
# LGTM

**Pull request merged**